app/views/plan/features.scala

```diff
@@ -14,74 +14,69 @@ object features:
 
   def apply()(using PageContext) =
     views.html.base.layout(
-      title = title,
+      title = trans.features.features.txt(),
       moreCss = cssTag("feature"),
       openGraph = lila.app.ui
         .OpenGraph(
-          title = title,
+          title = trans.features.features.txt(),
           url = s"$netBaseUrl${routes.Plan.features.url}",
-          description = "All of Lichess features are free for all and forever. We do it for the chess!"
+          description = trans.everybodyGetsAllFeaturesForFree.txt()
         )
         .some
     ) {
       main(cls := "box box-pad features")(
         table(
-          header(h1(dataIcon := licon.ScreenDesktop)("Website")),
+          header(h1(dataIcon := licon.ScreenDesktop)(trans.website())),
           tbody(
             tr(check)(
-              strong("Zero ads")
-            ),
-            tr(check)(
-              strong("No tracking")
+              strong(trans.features.zeroAdsAndNoTracking())
             ),
             tr(unlimited)(
-              "Play and create ",
-              a(href := routes.Tournament.home)("tournaments")
+              trans.features.playAndCreateX(
+              a(href := routes.Tournament.home)(trans.features.tournaments()))
```

Comment:
this is translation lego and it will break many languages. Do this instead:
```scala
a(href := routes.Tournament.home)(trans.features.playAndCreateTournaments()))
```
More occurences of this, below